README.md

```diff
@@ -180,17 +182,59 @@ Here are quick questions to help you determine which formatting to use, dependin
 |----------------|--------------------------------------|
 | Indicate the structure of the content | Blocks        |
 | Enter additional data/metadata        | Entities      |
-| Format a portion of the text          | Inline styles |
+| Format a portion of a line            | Inline styles |
 
-#### Blocks
+#### Custom blocks
 
 Simple blocks are very easy to create. Add a new block type to `blockTypes`, specifying which `element` and `className` to display the block as.
 
+Here is an example, creating a "Terms & conditions" block:
```

Comment:
It's not my fav example but I hope it's easy enough to relate to the need for this.